v21.2/deploy-app-netlify.md

```diff
@@ -0,0 +1,227 @@
+    This connection string is identical to the connection string you obtained earlier from the {{ site.data.products.db }} Console, with the exception of the database parameter (`leaderboard`).
+    {{site.data.alerts.end}}
+
+     By default, Prisma uses the local `DATABASE_URL` environment variable as the connection string to the database.
+
+1. Initialize Prisma Client:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ npx prisma generate
+    ~~~
+
+    Your application can now communicate with your CockroachDB cluster using Prisma Client.
+
+## Step 5. Deploy the application
+
+You can deploy web applications directly from GitHub to Netlify. In this tutorial, we use the [Netlify CLI](https://docs.netlify.com/cli) to deploy the app.
+
+1. Sign into your Netlify account:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ netlify login
+    ~~~
+
+1. Run the app server locally to preview your site:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ netlify dev
+    ~~~
+
+    If the local deployment succeeds, you should see the following message in your terminal:
+
+    ~~~
+    ┌─────────────────────────────────────────────────┐
+    │                                                 │
+    │   ◈ Server now ready on http://localhost:8888   │
+    │                                                 │
+    └─────────────────────────────────────────────────┘
+    ~~~
+
+    For a preview of the site, visit [http://localhost:8888](http://localhost:8888).
+
+1. From your repo directory, deploy your app with the Netlify CLI:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ netlify init
+    ~~~
+
+    Select the default options for each of the prompts. You will be required to authorize Netlify with GitHub.
+
+    After the app is deployed, you should see the following message:
+
+    ~~~
+    Success! Netlify CI/CD Configured!
+    ~~~
+
+1. In order for the deployed app to connect to your database, you need to define the `DATABASE_URL` environment variable in the production environment:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ netlify env:set DATABASE_URL $DATABASE_URL
+    ~~~
+
+    This command sets the remote `DATABASE_URL` environment variable for the site to the local `DATABASE_URL` environment variable that you set earlier to initialize Prisma.
+
+1. Navigate to the admin URL for your site:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ netlify open
+    ~~~
+
+    From the **Site overview** page, you can manage your site. The **Site overview** page also provides you with a public URL for your site.
+
+## See also
+
+- [How to build a Complete Webapp with React, TypeScript & CockroachDB](https://www.cockroachlabs.com/blog/react-typescript-cockroachdb-sample-app/#deploy-the-application-to-netlify)
+- [Create and Deploy an AWS Lambda Function Built on CockroachDB](deploy-lambda-function.html)
```

Comment:
I'm not convinced that this readership would appreciate Lambda and GCR; those seem like products for companies that have some in-house dev ops, whereas Netlify would be better if you don't have dev ops capabilities.